src/test/resources/tnt/v2/EventSchema.json

```diff
@@ -112,13 +112,13 @@
         },
         "otherFacility": {
           "id": "#/transportCall/properties/otherFacility",
-          "type": "string",
+          "type":  ["null","string"],
           "default": "",
           "maxLength": 6
         },
         "modeOfTransport": {
           "id": "#/transportCall/properties/modeOfTransport",
-          "enum":  [ "VESSEL", "RAIL", "TRUCK", "BARGE" ]
+          "type": ["null","string"]
```

Comment:
Don't we have the enum type?
...and null should not be allowed - it is a mandatory field...

Reply:
I am aware, but a null value is still received in the test data. I wanted to add a comment but cannot do in JSON file. 
I was thinking that we can add it as a bug? 
PS: many test cases fail if this specification is upheld

Reply:
Not quite sure what 

> ...but a null value is still received in the test data...

means... I hope you are creating test cases based on the API specs and not the test data :-D

Reply:
I do, I just diverted in this specific case. 
I can revert it and let the tests fail? or leave as is and create a bug issue for it? 

testdata includes ->> `modeOftransport: null`